alacritty/src/display.rs

```diff
@@ -627,25 +627,31 @@ impl Display {
         search_label: &str,
     ) -> String {
         // Add spacers for wide chars.
-        let mut text = String::with_capacity(search_regex.len());
+        let mut formatted_regex = String::with_capacity(search_regex.len());
```

Comment:
I've renamed all of the variables here to make them a little bit more clear.

The goal is that this will prevent accidentally using the incorrect variable in a follow up step or when renaming things and relying on shadowing. Now it should be very clear that `truncated_regex` for example is the search regex after truncation and before the label is prepended.